CONTRIBUTING.md

```diff
@@ -37,381 +48,295 @@ to sign the CLA. Therefore,
 
-  - Proficient in reviewing all aspects of writing quality, including grammar
-    and spelling, adherence to style guide conventions, organization, and
-    usability. Can coach newer writers to improve their contributions in these
-    areas.
+The
+[`/cherrypick` tool](https://github.com/kubernetes/test-infra/tree/master/prow/external-plugins/cherrypicker)
+can be use to automatically pull back changes from `master` to previous releases
+if necessary.
 
-- Primary reviewer for at least 10 substantial PRs to the docs, showing
-  substantial ability to coach for writing development.
+## Tools and Style
 
-- Reviewed or merged at least 30 PRs to the docs.
+Knative documentation follows the
+[Google Developer Documentation Style Guide](https://developers.google.com/style/).
+Use this as a reference for writing style questions.
 
-- Nominated by an area lead (with no objections from other leads).
+Knative uses several sets of tools to manage pull requests (PR)s and issues in a
+more fine-grained way than GitHub permissions allow. In particular, you'll
+regularly interact with
+[Prow](https://github.com/kubernetes/test-infra/tree/master/prow) to categorize
+and manage issues and PRs. Prow allows control of specific GitHub functionality
+without granting full "write" access to the repo (which would allow rewriting
+history and other dangerous operations). You'll most often use the following
+commands, but Prow will also chime in on most bugs and PRs with a link to all
+the known commands:
 
-  - Done through PR to update an OWNERS file.
+- `/assign @user1 @user2` to assign an issue or PR to specific people for review
+  or approval.
 
-### Responsibilities and privileges
+- `/lgtm` and `/approve` to approve a PR. Note that _anyone_ may `/lgtm` a PR,
+  but only someone listed in an `OWNERS` file may `/approve` the PR. A PR needs
+  both an approval and an LGTM — the `/lgtm` review is a good opportunity for
+  non-approvers to practice and develop reviewing skills. `/lgtm` is removed
+  when a PR is updated, but `/approve` is sticky — once applied, anyone can
+  supply an `/lgtm`.
 
-- Responsible for documentation quality control via PR reviews.
+- Both Prow (legacy) and GitHub actions (preferred) can run tests on PRs; once
+  all tests are passing and a PR has the `lgtm` and `approved` labels, Prow will
+  submit the PR automatically.
 
-  - Focus on long-term maintainability, adhering to style guide conventions,
-    overall information architecture, and usability from an engineering
-    standpoint.
+- You can also use Prow to manage labels on PRs with `/kind ...`,
+  `/good-first-issue`, or `/area ...`
 
-- Expected to be responsive to review requests as per
-  [community expectations](https://github.com/knative/community/blob/master/REVIEWING.md).
+- See [Branches](#branches) for details on the `/cherrypick` command.
 
-- Mentor members and contributors to improve their writing.
+# Engineer Guide
```

Comment:
This doesnt feel like the right categorization.

That said, the history and context for whats written below might be specific to "our early days"? Although i honestly dont know how many PRs we get today/recently for docs or code that dont belong in the knative/docs repo?

- This was basically an attempt to prevent PRs against the doc repo for "stuff" that belongs elsewhere (and reduce the unnecessary repo issue/pr maintenance overhead). 
- It also tries to articulate what types of user facing "code samples" we allow in the docs 

   ie. stuff we agree to own and maintain (ideally stuff that is generic and applies to all) VS stuff that is 
   platform specific and content that we do not maintain (ie. community members write and 
   maintain/update/fix it - we dont handle/deal with/chase down any  for those issues/PRs)

Reply:
I'm happy to remove this, this was basically a copy-over of the existing content here:

https://github.com/knative/docs/blob/master/CONTRIBUTING.md#putting-your-docs-in-the-right-place